src/transactions/TransactionMeta.cpp

```diff
@@ -407,7 +407,6 @@ OperationMetaBuilder::setLedgerChangesFromSuccessfulOp(
             {
                 changes.emplace_back(LEDGER_ENTRY_CREATED);
                 changes.back().created() = *le;
-                changes.back().created().lastModifiedLedgerSeq = ledgerSeq;
```

Comment:
I'm also wondering if we should update `lastModifiedLedgerSeq` when pushing into hotArchiveRestores and avoid the assignment above this. @SirTyson

Reply:
Hmm maybe not... `hotArchiveRestores` is expected to be the pre-state.

Reply:
I don't think there's a strong reason to maintain the original lastModifedLedgerSeq of the entry restored from the hot archive, as it shouldn't be used anywhere. I think the "pre-state" definition should be the state of the entry when it is originally restored, before the rest of the invocation makes any changes, not necessarily the state that is stored in the hot archive. Wrt to lastModifiedLedgerSeq, the pre sate actually is the current ledger, since we consider restores from the hot archive BucketList a write.